pkgs/applications/gis/qgis/ltr.nix

```diff
@@ -18,6 +18,7 @@ in symlinkJoin rec {
   pythonInputs = qgis-ltr-unwrapped.pythonBuildInputs ++ (extraPythonPackages qgis-ltr-unwrapped.py.pkgs);
 
   postBuild = ''
+    # unpackPhase
```

Comment:
What it is for?

Reply:
Just to sync `ltr.nix` with `default.nix`. This was the only difference except version and hash between those files.

Reply:
See: https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/gis/qgis/default.nix#L21